README.md

```diff
@@ -34,8 +34,8 @@ Multiple versions:
 
 ```yml
 steps:
-- name: Setup dotnet
 - uses: actions/checkout@v2
+- name: Setup dotnet
 - uses: actions/setup-dotnet@v1
```

Comment:
```suggestion
  uses: actions/setup-dotnet@v1
```

Reply:
Not sure I understand this suggestion. What did you change? 😄

Reply:
Ah, you removed the `-`. Good catch.